vcd/resource_vcd_rde.go

```diff
@@ -0,0 +1,435 @@
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The name of the Runtime Defined Entity",
+			},
+			"rde_type_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "The Runtime Defined Entity Type ID",
+			},
+			"external_id": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true, // It can be populated by a 3rd party
+				Description: "An external entity's ID that this Runtime Defined Entity may have a relation to",
+			},
+			"input_entity_url": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Description:  "URL that should point to a JSON representation of the Runtime Defined Entity and is used to initialize/override its contents",
+				ExactlyOneOf: []string{"input_entity_url", "input_entity"},
+			},
+			"input_entity": {
+				Type:                  schema.TypeString,
+				Optional:              true,
+				Description:           "A JSON representation of the Runtime Defined Entity that is defined by the user and is used to initialize/override its contents",
+				ExactlyOneOf:          []string{"input_entity_url", "input_entity"},
+				DiffSuppressFunc:      hasJsonValueChanged,
+				DiffSuppressOnRefresh: true,
+			},
+			"computed_entity": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "A computed representation of the actual Runtime Defined Entity JSON retrieved from VCD. Useful to see the actual entity contents if it is being changed by a third party in VCD",
+			},
+			"owner_user_id": {
+				Type:        schema.TypeString,
+				Description: "The ID of the user that owns the Runtime Defined Entity",
+				Computed:    true,
+			},
+			"org_id": {
+				Type:        schema.TypeString,
+				Description: "The organization of the Runtime Defined Entity",
+				Computed:    true,
+			},
+			"resolve": {
+				Type: schema.TypeBool,
+				Description: "If `true`, the Runtime Defined Entity will be resolved by this provider. If `false`, it won't be" +
+					"resolved and must be either done by an external component or with an update. The Runtime Defined Entity can't be" +
+					"deleted until the entity is resolved.",
+				Required: true,
+			},
+			"resolve_on_removal": {
+				Type: schema.TypeBool,
+				Description: "If `true`, the Runtime Defined Entity will be resolved before it gets deleted, to forcefully delete it." +
+					"Otherwise, destroy will fail if it is not resolved.",
+				Default:  false,
+				Optional: true,
+			},
+			"state": {
+				Type: schema.TypeString,
+				Description: "When created it will be in PRE_CREATED state. If the specified JSON in either `input_entity` or `input_entity_url` is correct, the state will be RESOLVED," +
+					"otherwise it will be RESOLUTION_ERROR. If an entity in an RESOLUTION_ERROR state, it will require to be updated to a correct JSON to be usable",
```

Comment:
```suggestion
					"otherwise it will be RESOLUTION_ERROR. If an entity is in a RESOLUTION_ERROR state, it will require to be updated to a correct JSON to be usable",
```

Reply:
Fixed (also in docs), thanks!